main.py

```diff
@@ -38,18 +35,35 @@ def inside_trades():
 
     buy, sell = transactions.get_trade_decisions()
     orders = helpers.place_trades(buy, sell)
-    helpers.activate_order(orders)
+    helpers.activate_orders(orders)
+
+
+def schedule_trades_for_year():
```

Comment:
You should pass the scheduler object as an input parameter here: 

`def schedule_trades_for_year(scheduler):`

Reply:
(Comment only applies if you decide that keeping this function is beneficial - see next comments)